src/list/item/list-item.test.tsx

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+import {render, fireEvent, screen} from "@testing-library/react";
+import "@testing-library/jest-dom";
+import userEvent from "@testing-library/user-event";
+
+import {testA11y} from "../../core/utils/test/testUtils";
+import ListItem, {ListItemProps} from "./ListItem";
+
+describe("<ListItem />", () => {
+  const defaultListItemProps: ListItemProps = {
+    testid: "list-item",
+    children: <p data-testid={"list-item.content"}>{"Test"}</p>
+  };
+
+  it("should render correctly", () => {
+    render(<ListItem {...defaultListItemProps} />);
+  });
+
+  it("should pass a11y test", async () => {
+    const {container} = render(<ListItem {...defaultListItemProps} />);
+
+    await testA11y(container, {rules: {listitem: {enabled: false}}});
+  });
+
+  it("should render children correctly", () => {
+    const {container} = render(<ListItem {...defaultListItemProps} />);
+
+    expect(container).toContainElement(screen.getByText("Test"));
+  });
+
+  it("Can be clicked", () => {
+    const handleClick = jest.fn();
+
+    const {container} = render(
+      <ListItem
+        clickableListItemProps={{onClick: handleClick, tabIndex: 2}}

```

Comment:
Why is the `tabIndex` `2`? Wouldn't it be enough if it was `0`?

Reply:
Default tabIndex is 0. It maybe not worth to test but I wanted to test that it also works when we give custom tabIndex
` tabIndex={clickableListItemProps.tabIndex || 0}`